<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nutzer_anlegen</class>
 <widget class="QDialog" name="nutzer_anlegen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="Eingabe_Benutzername">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
   <property name="placeholderText">
    <string>Maximal 32 Zeichen</string>
   </property>
  </widget>
  <widget class="QLabel" name="Name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Benutzername</string>
   </property>
  </widget>
  <widget class="QLabel" name="Profil">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nutzerprofil</string>
   </property>
  </widget>
  <widget class="QLabel" name="Passwort">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Passwort</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Eingabe_Passwort">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="placeholderText">
    <string>maximal 20 Zeichen</string>
   </property>
  </widget>
  <widget class="QLabel" name="Passwort_WDH">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Passwort wiederholen</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Eingabe_Passwort_WDH">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="Ueberschrift">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Neuen Benutzer anlegen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="Eingabe_Nutzerprofil">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Administrator</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Schreibberechtigt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Leseberechtigt</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="Button_abbrechen">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Abbrechen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Benutzer_anlegen">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>270</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Benutzer anlegen</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Eingabe_Benutzername</tabstop>
  <tabstop>Eingabe_Nutzerprofil</tabstop>
  <tabstop>Eingabe_Passwort</tabstop>
  <tabstop>Eingabe_Passwort_WDH</tabstop>
  <tabstop>Button_Benutzer_anlegen</tabstop>
  <tabstop>Button_abbrechen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
